### PR TITLE
Add elevation support for GPX files

### PR DESCRIPTION
**Pull request by jwenzler**

### CODE REVIEW

**Comment by robertleeplummerjr** [comment]
woot woot!

File: index.js [hunk 1]
```diff
@@ -86,6 +86,9 @@ function togpx( geojson, options ) {
           "name": options.featureTitle(f.properties),
           "desc": options.featureDescription(f.properties)
         };
+        if (coordinates[2]) {
```
Comment (tyrasd) [inline index.js]: This doesn't work when the z-coordinate is exactly 0 (or something else evaluating to false). Better to check against `!== undefined` or replace it with `coordinates.length > 2`.

Comment (jwenzler) [inline index.js]: Very good point, updated.  Thanks.

**Comment by tyrasd** [comment]
Thanks for updating the PR (also thanks for updating the tests)!

Thinking about it a bit more, I tend to prefer a solution that checks the z-coordinate against `undefined` because, theoretically, a GeoJSON [position](http://geojson.org/geojson-spec.html#positions) can contain additional elements after `[x,y,z]`. And if a user wants to include such data (e.g. a timestamp) but doesn't have altitudes, the coordinate would look like `[x,y, undefined, timestamp]` according to the specs.

Would you be so kind to update the PR, again? And it would be nice if you could [squash](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits) your commits into a single one.

**Comment by jwenzler** [comment]
Updated, thanks.

**Comment by tyrasd** [comment]
published as version `1.4.0`!

**Pull request merged**